.gitlab/kernel_matrix_testing/common.yml

```
@@ -332,8 +332,8 @@ notify_ebpf_complexity_changes:
             TEST_SET: no_usm
   allow_failure: true
   before_script:
-    - python3 -m pip install tabulate # Required for printing the tables
-    - python3 -m pip install -r tasks/libs/requirements-github.txt
+    - python3 -m pip install tabulate --break-system-packages# Required for printing the tables
```

```suggestion
    - python3 -m pip install tabulate --break-system-packages # Required for printing the tables
```
Otherwise this won't be recognized as a comment